frappe/public/js/frappe/ui/toolbar/navbar.html

```diff
@@ -16,7 +16,7 @@
 						id="navbar-search"
 						type="text"
 						class="form-control"
-						placeholder="{%= __("Search or type a command (Ctrl + G)") %}"
+						placeholder="{%= __("Search / Command / Recent (Ctrl + G)") %}"
```

Comment:
IMO the way to make this discoverable is to make it work first. There are 10 other undiscovered feature in awesomebar, and that's fine as long as they actually work. 

What we can change:
- When nothing is typed rank recent document entries higher and highlight somehow that those are recently visited documents. 
- Make recent document searchable by title too. Right now only name is searchable which is quite useless for many doctypes.

Reply:
@ankush I've created a new dropdown for recent and frequent docs and views. Seems better for discoverability. Also the recent and frequent docs and views together don't add up to too much, so I don't think they really need a search feature. Also got some inspiration from https://twitter.com/linear/status/1664349334844964866/photo/1.

And this is how it looks like:

![Screenshot 2023-08-25 at 3 21 17 AM](https://github.com/frappe/frappe/assets/2698932/fc911392-87f1-4cff-ba4f-b29c08dd36d1)

I've put some circle icon for now. If the dropdown is good to go, I'll need a clock icon @netchampfaris.

Also when we click the awesome bar now, only the awesome bar help thing is there. Is it a good idea to move it to the help section in the navigation (since users don't need to see the help thing every time they use the bar)?

Reply:
@anandbaburajan I don't think we should introduce another button in the navbar. Recent documents should be shown in the awesomebar when you haven't typed anything. This is how spotlight (macos) does it.

Reply:
> Recent documents should be shown in the awesomebar when you haven't typed anything.

@netchampfaris this is how it currently is and the problem with that is that many don't know about the feature. Hence I thought to separate it to a different section to make it more discoverable (also it doesn't need search since there aren't many things).

If we decide to keep them in the awesomebar itself, I'll insert the "recently/frequently viewed" labels into the awesomebar. Please share if you've any other ideas.

Reply:
> If we decide to keep them in the awesomebar itself, I'll insert the "recently/frequently viewed" labels into the awesomebar.

I don't think this is a problem of discoverability. I think it's safe to assume that people use awesomebar more than any other feature. To discover the recents feature, they just need to click it.

Also, UI is more about intuitiveness than correctness. This is a screenshot of Raycast that I use in place of spotlight. The label says "Search for apps and commands..." but it does a lot more than that. For e.g., it shows me suggestions of most used shortcuts.

<img width="776" alt="image" src="https://github.com/frappe/frappe/assets/9355208/fc86b460-09b1-4618-a716-d9a395d6efda">

Reply:
> To discover the recents feature, they just need to click it.

Me (and some other people) only realised recently that the things showing up in the awesome bar when it's clicked are the recent and frequent docs.

Reply:
Added labels for recently viewed and frequently viewed sections in awesomebar itself.